tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -257,4 +257,54 @@ public function testOnlyOwnCommentIsParsed(): void
             ->getColumn('col1')
             ->getComment());
     }
+
+    public function testUnnamedForeignKeyConstraintHandling(): void
```

Comment:
Would it be possible to move it to the abstract class and test across platforms? The more we have platform-specific tests, the lass consistent the abstraction is.

Reply:
The platform feature added only being used by sqlite, it would be skipped on all other platforms. The bug was an exception due to simple logic error, no need to make it more functional, execution of the sql is irrelevant for the fix, only not triggering the exception is, the test wouldnt need the assertions actually